pkg/R/group.R

```diff
@@ -0,0 +1,74 @@
+############################## GroupedData ########################################
+
+setClass("GroupedData",
+         slots = list(env = "environment",
+                      sgd = "jobj"))
+
+setMethod("initialize", "GroupedData", function(.Object, sgd) {
+  .Object@env <- new.env()
+  .Object@sgd <- sgd
+  .Object
+})
+
+groupedData <- function(sgd) {
+  new("GroupedData", sgd)
+}
+
+setMethod("count",
+          signature(x = "GroupedData"),
+          function(x) {
+            dataFrame(callJMethod(x@sgd, "count"))
+          })
+
+#' Agg
+#'
+#' Aggregates on the entire DataFrame without groups.
```

Comment:
Actually in this case it'll be good to write some documentation here to say what are the forms of aggregation supported. i.e. say something like We support two forms of aggregation 

``` 
df2 <- agg(df, <column> = <aggFunction>)
example: df2 <- agg(df, age = "sum") # Clarify what column name will get created here ?
or
df2 <- agg(df, newColName = aggFunction(column)) 
example: df2 <- agg(df, ageSum = sum(df$age)) # Creates a new column named ageSum
```